packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2688,6 +2699,19 @@ export interface _Identifiers {
     videoTile: string;
 }
 
+// @beta
+export interface ImageFileMetadata {
```

Comment:
I feel this one should extend FileSharingMetadata right?

Reply:
No unfortunately because we assign the `attachmentType` in `FileSharingMetadata` to be `fileSharing` this won't work.

Reply:
We cannot do that because `FileSharingMetadata` have `attachmentType` and when extending we cannot assign it a different value.
Originally we implemented them to extend from `BaseFileMetadata`, but it felt weird with the stable is using FileMetadata already so reverted the change.
```
interface BaseFileMetadata {
  name: string;
  url: string;
  extension: string;
}
```
We plan to add the base type and extend from it when the feature is closer to GA so stable will not feel odd.